docs/openapi.yaml

```diff
@@ -157,26 +157,16 @@ paths:
                   in: query
                   schema:
                     type: string
-                  examples:
-                    asc:
-                      summary: Ascending direction
-                      value: asc
-                    desc:
-                      summary: Descending direction
-                      value: desc
                 - name: search
                   in: query
-                  description: Field to filter products name
                   schema:
                     type: string
-                  example: Hexos
                 - name: productCategoryId
                   in: query
                   schema:
                     type: array
                     items:
                         type: string
-                    example: [1,2,3]
```

Comment:
Can we still include the deleted lines into the openapi.yaml? I guess it got deleted because of the generated proto

Reply:
sure mas, updated.